UnityProject/Assets/Scripts/Core/Input System/KeyboardInputManager.cs

```diff
@@ -236,9 +236,10 @@ private bool CheckComboEvent(KeyCombo keyCombo, KeyEventType keyEventType = KeyE
 		{ KeyAction.ChatLocal,		() => { ChatUI.Instance.OpenChatWindow(ChatChannel.Local); }},
 		{ KeyAction.ChatRadio,		() => { ChatUI.Instance.OpenChatWindow(ChatChannel.Common); }},
 		{ KeyAction.ChatOOC,		() => { ChatUI.Instance.OpenChatWindow(ChatChannel.OOC); }},
-		{ KeyAction.ToggleHelp,    () => { ChatUI.Instance.OnHelpButton(); }},
+		{ KeyAction.ToggleHelp,     () => { ChatUI.Instance.OnHelpButton(); }},
 		{ KeyAction.ToggleAHelp,    () => { ChatUI.Instance.OnAdminHelpButton(); }},
 		{ KeyAction.ToggleMHelp,    () => { ChatUI.Instance.OnMentorHelpButton(); }},
+		{ KeyAction.PushToTalk,     () => { VoiceChatManager.Instance.ClientPushToTalkPressed = !VoiceChatManager.Instance.ClientPushToTalkPressed; }},
```

Comment:
un a real push to talk, you have to hold the key to talk. This sounds more like a toggle.

Reply:
yeah,  sadly the current key bind manager isn't really set up for that, 
though there is UI to show when you microphone is active